mutt-forms/widget-personnummer
# feat: Handle protected identies

# Description

Adds handling of protected identities.

There is no flag in a response for a protected identity so we have to check certain fields for values. This means that the implementation is fragile should we change service provider or the fields could genuinely have no value.

## Type of change


# How Has This Been Tested?

Unit tests have been added.

To test in the Q/B app, you will need to npm link this package.

1. Proceed through the Q&B to the personnummer entry.
1. Enter a protected identity number, e.g. 194203052222
1. Click search
1. The message should be displayed that the identity is protected.

# Checklist:

- [x] Designer has reviewed the change
